src/common/libschedutil/alloc.c

```diff
@@ -33,24 +33,18 @@ int schedutil_alloc_request_decode (const flux_msg_t *msg,
 
 static int schedutil_alloc_respond (flux_t *h, const flux_msg_t *msg,
                                     int type, const char *note,
-                                    const char *annotations_json_str)
+                                    json_t *annotations)
```

Comment:
Since this refactors the code that was modified earlier, maybe ithe two libschedutil commits should be merged together?

Reply:
i think I see what you mean, some of the changes could be done in an earlier commit to avoid "churn".  Don't think they can be fully merged together though.

Reply:
Looking at the code I attached this to it may not be clear - I mainly meant the changes to the  `schedutil_alloc_respond()` internal function.